func-complete-check/check-marker/check-marker.v1.ts

```diff
@@ -0,0 +1,166 @@
+import * as RA from 'ramda-adjunct'
+import * as R from 'ramda'
+import { IAsyncTableService, AsyncTableService } from '../lib/storage-helper'
+import { ValidatedCheck } from '../typings/message-schemas'
+import moment from 'moment'
+import { ICheckFormService, CheckFormService } from '../lib/check-form.service'
+
+export interface ICheckMarkerFunctionBindings {
+  receivedCheckTable: Array<any>
+  checkNotificationQueue: Array<any>
+}
+
+declare interface Answer {
+  factor1: number
+  factor2: number
+  answer: string
+  sequenceNumber: number
+  question: string
+  clientTimestamp: string
+}
+
+declare interface CheckFormQuestion {
+  f1: number
+  f2: number
+}
+
+declare interface MarkingData {
+  answers: Array<Answer>
+  formQuestions: Array<CheckFormQuestion>
+  results: any
+}
+
+declare interface Mark {
+  mark: number
+  maxMarks: number
+  processedAt: Date
```

Comment:
I am surprised to maxMarks on a Mark object.  And that the question is not represented.  I will read on...

Reply:
the Mark object represents the overall mark.  is this a wording/naming issue?  regards answers - they will be stored row by row, likely in table storage.  although that is a separate decision

Reply:
as discussed, this is total mark.  answers will be persisted in separate ticket

Reply:
My bad.  Thanks for the explanation.